tests/samplers_tests/test_cmaes.py

```diff
@@ -324,19 +327,25 @@ def _create_trials() -> List[FrozenTrial]:
     ],
 )
 def test_sampler_attr_key(options: Dict[str, bool], key: str) -> None:
-    # Test sampler attr_key propery
+    # Test sampler attr_key propery.
```

Comment:
```suggestion
    # Test sampler attr_key property.
```
I guess what is meant here is 'property'

Reply:
Sorry I forgot to fix this typo. I fixed now